src/cli/common/config.ts

```diff
@@ -16,6 +16,7 @@ interface IUser {
   user: string;
   password: string;
   askForPassword?: boolean;
+  walletConnectString: string;
```

Comment:
Is this optional?

Reply:
Yes, is optional, if you set it it tries to read oracle credentials from whats, and if not set as before.